### PR TITLE
- added ArticleDto class

### PR DESCRIPTION
**Pull request by VadimKoziachiy**
- added ExtendedConversionService

### CODE REVIEW

File: src/main/java/news/blog/com/controller/ArticleController.java [hunk 1]
```diff
@@ -23,7 +26,7 @@ public String getArticles(Model model)
     @GetMapping("/{id}")
     public String getArticle(@PathVariable Long id, Model model)
     {
-        model.addAttribute("article", articleService.getArticle(id));
+        model.addAttribute("article", articleService.getArticleDto(id));
```
Comment (alxanko) [inline src/main/java/news/blog/com/controller/ArticleController.java]: let's name service methods w/o `Dto` entry

File: src/main/java/news/blog/com/service/impl/ArticleServiceImpl.java [hunk 2]
```diff
@@ -19,14 +24,21 @@
     }
 
     @Override
-    public ArticleEntity getArticle(Long id)
+    public ArticleDto getArticleDto(Long id)
     {
-            return articleRepository.findById(id).get();
+        return conversionService.convert(articleRepository.findById(id).orElseThrow(()-> new RuntimeException("Article not found " + id)), ArticleDto.class);
```
Comment (alxanko) [inline src/main/java/news/blog/com/service/impl/ArticleServiceImpl.java:31]: let's discuss Exception throwing this evening

File: src/main/java/news/blog/com/controller/ArticleController.java (hunk 1 above)
Comment (alxanko) [inline src/main/java/news/blog/com/controller/ArticleController.java]: let's develop our backend just as REST service w/o any views or frontend
model usage is unnecessary

Comment (VadimKoziachiy) [reply src/main/java/news/blog/com/controller/ArticleController.java]: done

File: src/main/java/news/blog/com/service/impl/ArticleServiceImpl.java (hunk 2 above)
Comment (VadimKoziachiy) [reply src/main/java/news/blog/com/service/impl/ArticleServiceImpl.java:31]: let's discuss

**Pull request merged**